src/comp/comp.py

```diff
@@ -21,51 +20,68 @@ def __repr__(self):
     Human("David", 31),
 ]
 
-# Write a list comprehension that creates a list of names of everyone
-# whose name starts with 'D':
+# name starts with 'D':
 print("Starts with D:")
 a = []
+for x in humans:
+    if x.name.startswith("D"):
+        a.append(x.name)
```

Comment:
For all of these problems, you can do them in 1 line via list comprehensions.

so instead of the for loop with some logic inside of it, we can write
`a = [x.name for x in humans if x.name.startswith("D")]`

I think they are covered in the first lecture from Monday, but the general idea is
` x = [expression        loop         condition]`
and just lets us pack the contents of a larger loop into a single list comprehension.